packages/grid/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -353,10 +353,6 @@ export const GridRootStyles = styled('div', {
       display: 'flex',
       boxShadow: theme.shadows[2],
       backgroundColor: (theme.vars || theme).palette.background.paper,
-      '&:focus-within': {
```

Comment:
Cell focus seems broken on nested grids in https://deploy-preview-7111--material-ui-x.netlify.app/x/react-data-grid/demo/
Should we update the demo to use the new class name?

```diff
diff --git a/docs/data/data-grid/demo/PopularFeaturesDemo.tsx b/docs/data/data-grid/demo/PopularFeaturesDemo.tsx
index f439522..49b04c3 100644
--- a/docs/data/data-grid/demo/PopularFeaturesDemo.tsx
+++ b/docs/data/data-grid/demo/PopularFeaturesDemo.tsx
@@ -393,7 +393,7 @@ export default function PopularFeaturesDemo() {
           [`& .${gridClasses.detailPanel}`]: {
             background: 'transparent',
           },
-          [`& .${gridClasses.cell}:focus, & .${gridClasses.cell}:focus-within`]: {
+          [`& .${gridClasses['cell--outlined']}`]: {
             outline: 'none',
           },
           [`& .${gridClasses.columnHeader}:focus, & .${gridClasses.columnHeader}:focus-within`]:
```